garrulous-gobies/Bangazon-sprint-2
# Nl pay type routing

## Link to Ticket
Closes # [https://github.com/garrulous-gobies/Bangazon-sprint-2/issues/63](63)

## Description of Proposed Changes
- added a hidden input on the add payment form to capture the next parameter from the url query string
- added the query string of the origin url to the add payment form
-form submission redirects to 'next' query string

## Steps to Test

```
git fetch --all
git checkout nl-pay-type-routing

```
navigate to add payment form from profile and complete order screens and ensure proper rerouting

## Impacted Areas in Application

List general components of the application that this PR will affect:

* Profile
*Complete_order
*add_payment

## Mentions @username

Tag users that need to review this code
